idd/Energy+.idd.in

```diff
@@ -73676,6 +73676,8 @@ Chiller:Electric:EIR,
        \note corresponds to this fraction multiplied by the maximum condenser flow rate. This input is only used
        \note when the "Condenser Flow Control" input is set to "ModulatedChillerPLR", "ModulatedLoopPLR" or
        \note "ModulatedDeltaTemperature".
+       \minimum> 0.0
+       \default 0.2
```

Comment:
Thanks @lymereJ 

Just checking: is 0.0 not a valid value?

Reply:
And does > 1 make sense here? Otherwise `\maximum 1` would be good?

Reply:
The minimum was copy/paste typo, good catch. I'll add the maximum.